test/client.spec.ts

```diff
@@ -36,11 +37,25 @@ const dnsPort = parseInt(process.env.CERT_TEST_PORT || '0', 10);
 const domain = process.env.CERT_TEST_DOMAIN || '';
 const email = process.env.CERT_TEST_EMAIL || '';
 
+// Utility function for removing certificate directories
+function removeDirectory(dir: string): Bluebird<void> {
+	return Bluebird.fromCallback((cb: (err: Error, result?: any) => void) => {
+		exec(`rm -rf ${dir}`, cb);
```

Comment:
nit: I don't like `rm -rf` as it makes a hard requirement on the base OS. `rimraf` might be nicer. Also, there is no check on `dir` to make sure it is not `/` etc, but I appreciate this might be more work than is necessary.

Reply:
Yeah, I had considered this at the time, although the tests can only run on a privileged machine anyway. I'll rework a bit.